src/deployment.ts

```diff
@@ -87,6 +87,32 @@ export class Deployment {
         return Promise.resolve();
     }
 
+    @vsCommand('openshift.deployment.scale')
+    static async scalingPod(component: KubernetesObject): Promise<void> {
+        if (!component) {
+            return;
+        }
+        let count = '1';
```

Comment:
Do we store somewhere or can we find out somehow and use the current scale number as an initial value?